tests/python/quantize/test_pass_quantize.py

```diff
@@ -0,0 +1,80 @@
+import math
+import numpy as np
+import tvm
+from tvm import relay
+from tvm.relay import quantize as qtz
+
+def test_simulated_quantize():
+    data = relay.var("data", relay.ty.TensorType((3, 4, 5, 6), "float32"))
+    out = qtz._annotate.attach_simulated_quantize(data, 1)
+    out = relay.ir_pass.infer_type(out)
+    assert out.checked_type == out.args[0].checked_type
+    assert out.args[1].checked_type == relay.ty.TensorType(tuple(), "float32")
+    assert out.args[2].checked_type == relay.ty.TensorType(tuple(), "int32")
```

Comment:
When I dived into this PR, this assertion fails:
`out.args[2].checked_type` is `TensorType([], float32)`, instead of `TensorType([], int32)`